gcs/src/gcs.cpp

```diff
@@ -993,6 +1037,23 @@ gcs_handle_flow_control (gcs_conn_t*                conn,
         gcs_sm_continue (conn->sm); // last CONT request
     }
 
+    if (conn->params.fc_auto_evict_window > 0) {
+        const long long paused =
+            gcs_sm_paused_in_window_get(conn->sm, conn->fc_auto_evict_window_ns,
+                                        !(conn->local_act_id % 10)); // erase not very often
+        gu_debug ("FC auto evict: paused for %lld ns.", paused);
+
+        if (paused >= conn->fc_auto_evict_threshold_ns) {
+            gu_info ("FC auto evict: pause limit exceeded (%f from %f sec. in %f sec. window). "
```

Comment:
Can we rephrase this message to something readable like
```
 Triggering automatic eviction of the node since flow control pause limit exceeded from %f to %f in %f sec window. The node will be aborted.
```

Reply:
Donw